app/models/manager_refresh/dto_collection.rb

```diff
@@ -24,6 +25,10 @@ def initialize(model_class, manager_ref: nil, association: nil, parent: nil, str
       @attributes_whitelist  = Set.new
       @custom_save_block     = custom_save_block
       @internal_attributes   = [:__feedback_edge_set_parent]
+      @complete              = complete.nil? ? true : complete
+
+      blacklist_attributes!(attributes_blacklist) if attributes_blacklist.present?
```

Comment:
hm, yea, there are synonyms, at first I had there include/exclude_attributes

Not sure what is the best :-)